packages/botonic-plugin-flow-builder/src/utils.ts

```diff
@@ -51,3 +52,24 @@ function resolveObjectKey(object: any, key: string): any {
 export function inputHasTextData(input: Input): boolean {
   return input.data !== undefined && input.type === INPUT.TEXT
 }
+
+function isNluAllowed(
+  request: ActionRequest,
+  nluFlag: 'allowKeywords' | 'allowSmartIntents' | 'allowKnowledgeBases'
```

Comment:
Suggestion: I would try to use the same keys as `InShadowingConfig`, what do you think?
```suggestion
  nluFlag: keyof InShadowingConfig
```